src/components/Preferences/BodyApplication.vue

```diff
@@ -34,22 +44,23 @@ export default Vue.extend({
   <tabbed
     v-bind="$attrs"
     class="action-tabs"
+    :class="{ 'tab-mute': isPlatformWindows }"
     :no-content="true"
     @changed="tabSelected"
   >
     <tab
       label="Environment"
```

Comment:
I was thinking this would still have `v-id="!isPlatformWindows"`, but I guess this is fine too?

Reply:
I think that ends up being a little redundant. We've essentially wrapped our tabs in `v-show` with the `tab-mute` class (with the added benefit of not showing other tab decorations that we wouldn't get otherwise)

I'm thinking there's either upstream changes that need to be made to the tabbed component so that this use case would work a little more cleanly. Alternatively, maybe we need to come up with a better pattern to not use the tabbed component when we don't need tabs..